src/cr/cube/cube.py

```diff
@@ -779,17 +814,34 @@ def _flat_values(self):
         )
 
 
+class _SumMeasure(_BaseMeasure):
+    """Statistical sum values from a cube-response."""
+
+    @lazyproperty
+    def missing_count(self):
+        """numeric representing count of missing rows reflected in response."""
+        return self._cube_dict["result"]["measures"]["sum"].get("n_missing", 0)
+
+    @lazyproperty
+    def _flat_values(self):
+        """Return tuple of mean values as found in cube response.
+        Mean data may include missing items represented by a dict like
+        {'?': -1} in the cube response. These are replaced by np.nan in the
+        returned value.
+        """
+        return tuple(
+            np.nan if type(x) is dict else x
+            for x in self._cube_dict["result"]["measures"]["sum"]["data"]
+        )
```

Comment:
btw, I believe Greg is doing this in his branch, since he added something to do with flat values, so you might want to take a look at his branch before you merge it in, to get a sense of where the conflicts might lie.